.flowconfig

```diff
@@ -18,3 +18,7 @@
 [options]
 
 [strict]
+
+[untyped]
+# React-beautiful-dnd contains a typing error --> ignore it
```

Comment:
I'm not sure what's the difference between `untyped` and `ignore` section but I would just add `react-beautiful-dnd` to the ignore section.
Do you know the difference between them?

Reply:
OK, in the [docs](https://flow.org/en/docs/config/untyped/) the difference is mentioned and I think it's OK to keep React-beautiful-dnd as I think it's better to have it in this section so import/require can resolve it.